File: src/main/java/com/hopper/model/availability/agoda/PolicyTranslated.java
package com.hopper.model.availability.agoda;

import javax.xml.bind.annotation.*;

@XmlRootElement(name="PolicyTranslated")
@XmlAccessorType(XmlAccessType.FIELD)
public class PolicyTranslated {

    @XmlAttribute
    private String language;

    @XmlValue
    private String content;

    public String getLanguage() {
        return language;
    }

    public void setLanguage(String language) {
        this.language = language;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    @Override
    public String toString() {
        return "PolicyText{" +
                "language='" + language + '\'' +
                ", content='" + content + '\'' +
                '}';
    }
}

File: src/main/scala/com/hopper/model/availability/agoda/response/PolicyTranslated.scala
package com.hopper.model.availability.agoda.response

import javax.xml.bind.annotation.{XmlAccessType, XmlAccessorType, XmlAttribute, XmlRootElement, XmlValue}

@XmlRootElement(name = "PolicyTranslated")
@XmlAccessorType(XmlAccessType.NONE)
class PolicyTranslated
{
    @XmlAttribute(name = "language")
    var language: String = _

    @XmlValue
    var content: String = _
}
